apps/verticalhome/test/marionette/app_packaged_resume_test.js

```diff
@@ -13,24 +13,23 @@ marionette('Vertical Home - Packaged App Resuming Downloads', function() {
   var client = marionette.client(require(__dirname + '/client_options.js'));
 
   var server;
+  var subject;
+  var system;
+  var appInstall;
   setup(function(done) {
     var app = __dirname + '/fixtures/template_app';
     createAppServer(app, client, function(err, _server) {
       server = _server;
-      done(err);
-    });
-  });
 
-  var subject;
-  var system;
-  var appInstall;
-  setup(function() {
-    subject = client.loader.getAppClass('verticalhome');
-    system = client.loader.getAppClass('system');
-    appInstall = new AppInstall(client);
+      subject = client.loader.getAppClass('verticalhome');
```

Comment:
if your going to edit tests here too can you do it in another commit (better invidual commits per test so it's easy for the owners to figure this bit out)?